tests/acceptance/browzine-summon-adapter-test.js

```diff
@@ -649,5 +750,230 @@ describe("BrowZine Summon Adapter >", function() {
+                "url_for_landing_page": "http://hdl.handle.net/2445/147225",
+                "url_for_pdf": "http://diposit.ub.edu/dspace/bitstream/2445/147225/1/681991.pdf",
+                "version": "acceptedVersion"
+              }
+            })
+          });
+
+          var template = documentSummary.find(".browzine");
+
+          expect(template).toBeDefined();
+
+          expect(template.text().trim()).toContain("Article PDF: Download Now (Accepted Manuscript via Unpaywall)");
+          expect(template.find("a.unpaywall-manuscript-article-pdf-link").attr("href")).toEqual("http://diposit.ub.edu/dspace/bitstream/2445/147225/1/681991.pdf");
+          expect(template.find("a.unpaywall-manuscript-article-pdf-link").attr("target")).toEqual("_blank");
+          expect(template.find("img.browzine-pdf-icon").attr("src")).toEqual("https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg");
+        });
+      });
+
+      describe("unpaywall best open access location host type repository and version acceptedVersion and does not have a pdf url >", function() {
+        it("should enhance the article with an unpaywall manuscript article link", function() {
+          var request = jasmine.Ajax.requests.mostRecent();
+
+          request.respondWith({
+            status: 200,
+            contentType: "application/json",
+            response: JSON.stringify({
+              "best_oa_location": {
+                "endpoint_id": null,
+                "evidence": "oa repository (via pmcid lookup)",
+                "host_type": "repository",
+                "is_best": true,
+                "license": null,
+                "pmh_id": null,
+                "repository_institution": null,
+                "updated": "2020-02-22T01:10:19.539950",
+                "url": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041472-do-not-use",
+                "url_for_landing_page": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041472",
+                "url_for_pdf": null,
+                "version": "acceptedVersion"
+              }
+            })
+          });
+
+          var template = documentSummary.find(".browzine");
+
+          expect(template).toBeDefined();
+
+          expect(template.text().trim()).toContain("Article Link: Read Article (Accepted Manuscript via Unpaywall)");
+          expect(template.find("a.unpaywall-manuscript-article-link").attr("href")).toEqual("https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041472");
+          expect(template.find("a.unpaywall-manuscript-article-link").attr("target")).toEqual("_blank");
+          expect(template.find("img.browzine-article-link-icon").attr("src")).toEqual("https://assets.thirdiron.com/images/integrations/browzine-article-link-icon.svg");
+        });
+      });
+
+      describe(`unpaywall best open access location host type repository and version null and has a pdf url from nih.gov or europepmc.org >`, function() {
```

Comment:
I'm thinking of two more tests that I don't think we have right now:

1. What happens when Unpaywall has no best open access location? (we wouldn't enhance the article - [Nomad test here] )(https://github.com/thirdiron/nomad/blob/develop/test/unpaywall.test.js#L50-L63)
2. What happens when Unpaywall has a best open access location, but the data is something that we cannot use to enhance the article? (we wouldn't enhance the article - [Nomad test here](https://github.com/thirdiron/nomad/blob/develop/test/unpaywall.test.js#L236-L251))

I think both of these will be pretty useful tests to have for Summon and Primo.

Reply:
Hi @karlbecker! 

Added the check for when we have no best open access location: https://github.com/thirdiron/browzine-discovery-service-adapters/pull/69/commits/c542ebdbe6f48fcf4c85178196d442f4fbe81614

But why are we testing for `submittedVersion` when we don't have any code that looks for it or any could that would break if it were present (again, since we don't look for it at all)? https://github.com/thirdiron/nomad/blob/develop/test/unpaywall.test.js#L236-L251

Reply:
Thanks for asking @tiandavis - it looks like the reason Nomad had it was because of [this diff](https://github.com/thirdiron/nomad/commit/28ca0d878fa6b50f8836d5ede7b104b37a394da5#diff-1741974c2495e0b31a466ba29a71357aL189). We used to do something with submittedVersion, then we changed how it worked, and I just left the test in to confirm that we do nothing with `submittedVersion`.

So I think the only argument I can come up with is if we want to specify what, if anything, we expect for `submittedVersion`.

But yeah, since we don't have it anywhere in this code, nor any history         about it in this, maybe just forget that test?

Reply:
Thinking we should pass for now.